packages/driver/src/cy/commands/querying.js

```diff
@@ -246,6 +274,26 @@ module.exports = (Commands, Cypress, cy, state) => {
             return requests
           }
 
+          if (['route2', 'intercept'].includes(command.get('name'))) {
+            const requests = getAliasedRequests(alias, state)
+            const specifier = /\.(all|[\d]+)$/.exec(selector)
```

Comment:
ughh, it would be nice to comment or extract this

Reply:
```suggestion
            // detect alias.all and alias.index
            const specifier = /\.(all|[\d]+)$/.exec(selector)
```